server/src/test/java/org/elasticsearch/common/lucene/uid/VersionsTests.java

```diff
@@ -197,8 +197,8 @@ public void testLuceneVersionOnUnknownVersions() {
                 version.luceneVersion);
 
         // between two known versions, should use the lucene version of the previous version
-        version = Version.fromString("6.2.50");
-        assertEquals(VersionUtils.getPreviousVersion(Version.V_6_2_4).luceneVersion, version.luceneVersion);
+        version = Version.fromString("8.0.50");
+        assertEquals(Version.V_8_0_0.luceneVersion, version.luceneVersion);
```

Comment:
Can we make this not depend on any constants? Perhaps use `VersionUtils.getPreviousVersion(Version.CURRENT).toString() + "1"`, so the bugfix portion will be past any known versions, but still before current.